demo/src/main/java/com/demo/DemoApplication.java

```diff
@@ -0,0 +1,49 @@
+package com.demo;
+
+import com.demo.model.Book;
+import com.demo.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class DemoApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
Use lombok @requiredargsconstructor instead of @autowired

Reply:
refactored